core/ui/src/main/kotlin/com/merxury/blocker/core/ui/component/ComponentList.kt

```diff
@@ -59,6 +70,7 @@ fun ComponentList(
     isSelectedMode: Boolean = false,
     onSelect: (ComponentInfo) -> Unit = {},
     onDeselect: (ComponentInfo) -> Unit = {},
+    onRefresh: () -> Unit = {},
```

Comment:
Same as above, the refresh feature does not belong to the list itself. Consider moving this feature to the screen level.